src/client/utils/campaignCode.ts

```diff
@@ -9,7 +9,7 @@ const getPersistedCampaignCode = () => {
 }
 
 const saveCampaignCode = (campaignCode: string) => {
-  cookieStorage.setItem(CAMPAIGN_CODE_COOKIE_KEY, campaignCode)
+  cookieStorage.setItem(CAMPAIGN_CODE_COOKIE_KEY, campaignCode, { path: '/' })
```

Comment:
Does `{ path: '/' }` make the cookie availability through the whole app (every subdoman)? 🤔

Reply:
I don't think it applies to subdomains but the Cookie header will be sent on all URL paths:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#path_attribute

This is just how it was setup before, that's why I included it.